chat-widget/src/components/headerstateful/interfaces/IHeaderStatefulParams.ts

```diff
@@ -13,6 +13,18 @@ export interface IHeaderStatefulParams {
      */
     outOfOfficeHeaderProps?: IHeaderProps;
 
+   /**
+    * isEmailTranscriptDisabled: Props to check if Email Transcript is Disabled
+    * These props are used for maintaining consistent behaviour between header and footer
```

Comment:
we shouldn't bring footer props to header; we need figure out a better design

Reply:
How about we have a separate boolean prop for header like:

```
    // Header prop to check if download transcript is enabled
    const shouldShowDownloadChatCopy = !(emailTranscriptDisabled && downloadTranscriptDisabled);
```

Reply:
Isn't confirmation pane capable to handle this message based on footer logic? All we had to do show the footer.